wct.conf.js

```diff
@@ -1,21 +0,0 @@
-module.exports = {
-  suites: ['.test/'],
```

Comment:
We need this file for at least the `suites` param as by default `wct` looks for tests in `test` but we run in `.test/` as we transpile.